### PR TITLE
Consider branch when generating transaction ID

### PR DESCRIPTION
**Pull request by DBoag**

### CODE REVIEW

**Comment by DBoag** [comment]
This PR is for this issue: https://github.com/drachtio/drachtio-server/issues/261

**Review by davehorton — APPROVED** [review]

File: src/drachtio.cpp [hunk 1]
```diff
@@ -208,6 +208,10 @@ namespace drachtio {
           sip->sip_cseq->cs_method_name) ;
         str.append("|") ;
         str.append( boost::lexical_cast<std::string>(sip->sip_cseq->cs_seq) ) ;
+        if (sip->sip_via && sip->sip_via->v_branch) {
```
Comment (davehorton) [inline src/drachtio.cpp:211]: I'm wondering, is there an expected case where this is false?  I suppose we could get a request from a very old client that didn't add a branch (?)

**Comment by davehorton** [comment]
This commit actually breaks sip proxy functionality.  I will check in what I think is the correct fix shortly in a different PR.

**Comment by davehorton** [comment]
I've merged my fix for this issue in https://github.com/drachtio/drachtio-server/pull/282

**Pull request closed without merging**